tests/examples/tokens/test_erc1155.py

```diff
@@ -109,10 +110,12 @@ def test_pause(erc1155, w3, assert_tx_failed):
     )
 
     # check safetransferfrom and safebatchtransferfrom
-    assert_tx_failed(lambda: erc1155.safeTransferFrom(a1, a2, 21, 1, "", transact={"from": a1}))
+    assert_tx_failed(
+        lambda: erc1155.safeTransferFrom(a1, a2, 21, 1, DUMMY_BYTES_DATA, transact={"from": a1})
```

Comment:
Why do these tests need to change?

Reply:
`web3` raises an error where it is unable to find a matching function signature - it treats `""` as `string` type, and `b""` as `bytes` type, instead of `bytes32`.

```
>           raise Web3ValidationError(message)
E           web3.exceptions.Web3ValidationError: 
E           Could not identify the intended function with name `safeTransferFrom`, positional arguments with type(s) `address,address,int,int,str` and keyword arguments with type(s) `{}`.
E           Found 1 function(s) with the name `safeTransferFrom`: ['safeTransferFrom(address,address,uint256,uint256,bytes32)']
E           Function invocation failed due to no matching argument types.
```

```
>           raise Web3ValidationError(message)
E           web3.exceptions.Web3ValidationError: 
E           Could not identify the intended function with name `safeTransferFrom`, positional arguments with type(s) `address,address,int,int,bytes` and keyword arguments with type(s) `{}`.
E           Found 1 function(s) with the name `safeTransferFrom`: ['safeTransferFrom(address,address,uint256,uint256,bytes32)']
E           Function invocation failed due to no matching argument types.
```

Reply:
ah ok, maybe call it DUMMY_BYTES32_DATA then